sdk/core/core-auth/review/core-auth.api.md

```diff
@@ -22,6 +22,14 @@ export interface GetTokenOptions {
 // @public
 export function isTokenCredential(credential: any): credential is TokenCredential;
 
+// @public
+export class RawTokenCredential implements TokenCredential {
+    constructor(token: string, expiresOn?: Date);
+    expiresOn: Date;
+    getToken(_scopes: string | string[], _options?: GetTokenOptions): Promise<AccessToken | null>;
```

Comment:
Why are the params having underscore here?

Reply:
They are unused in the scope of this function, adding the `_` prefix tells the linter to ignore the fact that they aren't used.  I can look into using an eslint rule disable comment instead.

Reply:
nah, not worth it
this is fine